docs/DeveloperGuide.md

```diff
@@ -323,7 +324,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `*`      | experienced social worker                 | locate a link where I can share my views and suggestions on the app        | help the app be more user centered and helpful over time                                               |
 | `*`      | efficient person                          | create visit paths based on proximity of beneficiaries                     | to be able to serve the most beneficiaries a day                                                       |
 | `*`      | social person                             | add notes of on conversations with beneficiaries                           | develop stronger relationships by building rapport                                                     |
-
+| `**` | new user | see a sample of the product features on display | so that I know how a feature can be used to maximise the value |
```

Comment:
Do you think it would be important to insert this into its sorted order within the user stories?

Reply:
right. thanks for pointing it out!